src/tools/id-mapping/components/results/__tests__/IDMappingResult.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import { fireEvent, screen } from '@testing-library/react';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+import customRender from '../../../../../shared/__test-helpers__/customRender';
+import IDMappingResult from '../IDMappingResult';
+
+import SimpleMappingData from '../__mocks__/SimpleMapping';
+import SimpleMappingDetails from '../__mocks__/SimpleMappingDetails';
+import UniProtkbMapping from '../__mocks__/UniProtkbMapping';
+import UniProtkbMappingDetails from '../__mocks__/UniProtkbMappingDetails';
+
+import { ViewMode } from '../../../../../shared/components/results/ResultsData';
+
+const mock = new MockAdapter(axios);
```

Comment:
Non-critical and can be done at a later time: can combine all of these mock statements as is done in https://github.com/ebi-uniprot/uniprot-website/blob/de90a1a8b66ba42d4c65b7b57ef29cd1b74b8e59/src/uniprotkb/components/entry/__tests__/Entry.spec.tsx#L34